cypress/integration/db.spec.js

```diff
@@ -0,0 +1,112 @@
+/// <reference types="Cypress" />
+
+describe('DB Testing', function () {
+  let info = {}
+  // let ephemeralLoginInfo = {}
+
+  beforeEach(() => {
```

Comment:
Check [this approach](https://github.com/encrypted-dev/userbase/commit/9f132a8c5b488bef76e06a71efa9c8babc818e99#diff-bb3f886dfa503651bc3390eed0f24f6eR2-R12) out. I think a before hook is good enough (rather than beforeEach) + you can set a higher scoped userbase variable [once](https://github.com/encrypted-dev/userbase/commit/9f132a8c5b488bef76e06a71efa9c8babc818e99#diff-bb3f886dfa503651bc3390eed0f24f6eR7) inside that before hook (rather than grab it from the response to the `cy.window()` promise each test)